com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/SettingsService.java

```diff
@@ -0,0 +1,8 @@
+package com.ca.lsp.cobol.service;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public interface SettingsService {
```

Comment:
Could you, please, provide also javadocs for this interface and its implementation?